tests/utils_tests/test_html.py

```diff
@@ -422,6 +425,78 @@ def test_urlize(self):
             with self.subTest(value=value):
                 self.assertEqual(urlize(value), output)
 
+    def test_urlize_assume_https(self):
+        """
+        urlize() uses https:// as default protocol when URLIZE_ASSUME_HTTPS is True.
+        """
```

Comment:
```suggestion
    def test_urlize_https_default_protocol(self):
```
I think let's remove the docstring as it's one less thing to update when the deprecation ends